test/TTBasic/rlexe.xml

```diff
@@ -5,471 +5,471 @@
       <files>accessor.js</files>
       <compile-flags>-TTRecord:!accessorTest -TTSnapInterval:0</compile-flags>
       <baseline>accessorRecord.baseline</baseline>
-      <tags>exclude_dynapogo</tags>
+      <tags>exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized</tags>
```

Comment:
Instead of adding the tag to each test, add the tag to exclude the TTBasic and TTExecuteBasic directory in  test\rlexedirs.xml